flaskbb/forum/views.py

```diff
@@ -276,6 +332,7 @@ def get(self, forum_id, slug=None):
             forumsread=forumsread,
         )
 
+    # TODO(anr): Clean this up. @_@
```

Comment:
this method is a mess i know :see_no_evil:

Reply:
:speak_no_evil: